### PR TITLE
Address API Review feedback

### PR DESCRIPTION
**Pull request by bricelam**
Part of #15662

### CODE REVIEW

**Comment by bricelam** [comment]
Note, @AndriySvyryd has more pending in his [Issue214_Part5](https://github.com/aspnet/EntityFrameworkCore/compare/Issue214_Part5) branch.

File: src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj [hunk 1]
```diff
@@ -52,15 +52,23 @@ Microsoft.Data.Sqlite.SqliteTransaction</Description>
       <AutoGen>True</AutoGen>
       <DependentUpon>Resources.Designer.tt</DependentUpon>
     </Compile>
+    <Compile Update="SqliteCommand.cs">
```
Comment (AndriySvyryd) [inline src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj]: 🛑

File: src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj [hunk 2]
```diff
@@ -52,15 +52,23 @@ Microsoft.Data.Sqlite.SqliteTransaction</Description>
...
       <DependentUpon>SqliteConnection.CreateAggregate.tt</DependentUpon>
+      <SubType>Component</SubType>
     </Compile>
     <Compile Update="SqliteConnection.CreateFunction.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
       <DependentUpon>SqliteConnection.CreateFunction.tt</DependentUpon>
+      <SubType>Component</SubType>
+    </Compile>
+    <Compile Update="SqliteConnection.cs">
```
Comment (AndriySvyryd) [inline src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj]: 🛑

File: src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj (hunk 1 above)
Comment (ajcvickers) [reply src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj]: Does VS do this automatically? If so, is there any point fighting it?

Comment (bricelam) [reply src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj]: Don't fight it. We'll change them to `Code` to make the dev experience better as soon as the project system stops stomping over it.

**Pull request merged**